Source/Core/Core/HW/WiimoteReal/WiimoteReal.cpp

```diff
@@ -24,6 +25,7 @@
 #include "Core/HW/Wiimote.h"
 #include "Core/HW/WiimoteCommon/DataReport.h"
 #include "Core/HW/WiimoteCommon/WiimoteHid.h"
+#include "Core/HW/WiimoteEmu/MotionPlus.h"
```

Comment:
The only thing is that now this include looks out of place, perhaps a comment would be good.

Reply:
I don't agree.